src/index.js

```diff
@@ -146,9 +152,9 @@ export default class Pipeliner {
 
   /**
    * Runs the specified pipeline, passing the arguments to each task.
-   * @param  {string}    pipeline the pipeline to run
+   * @param  {string}    pipeline The name of the pipeline to run
    * @param  {...object} args     Arguments to pass to the pipeline tasks
-   * @return {Promise}             A promise that is executed when the pipeline completes.
+   * @return {Promise}            A promise that is fulfilled when the pipeline completes; it is rejected if any task in the pipeline fails (throws an error or returns a promise that is rejected)
```

Comment:
Okay, bear with me; I'm still trying to absorb all the Bluebird promise extensions. Let me see if I understand this bit of code:
```
    return Promise.mapSeries(_jobs, (job) => {return Promise.resolve(job(...args))})
```
For each job in the series, first the job is evaluated – its return value is passed to Promise#resolve, but any exception passes up to mapSeries. The way you get serialization of asynchronous tasks here is for `job` to return a promise, which will in turn be tracked by the promise returned by Promise#resolve, which will be serialized by mapSeries. 
The promise returned by mapSeries (and in turn by Pipeliner#run) fulfills when the series completes (with an array of values from the jobs); it is rejected if any job returns a promise that is rejected. 
I couldn't tell from the Bluebird docs what happens if the iterator passed to mapSeries throws an error. I was guessing it caught it, but from your comment it sounds like it doesn't. 
So, if I understand this correctly, that behavior ends up being a bit strange. If any job fails synchronously (the job throws an exception when evaluated), then Pipeliner#run throws an exception. But if any job fails asynchronously, that's "captured" in the returned promise. 
That seems a bit squirrelly; I would think you'd want to capture those synchronous exceptions in the returned promise, too.